src/renderer/components/cluster-manager/lens-views.ts

```diff
@@ -39,6 +37,8 @@ export async function initView(clusterId: ClusterId) {
   }, { once: true });
   lensViews.set(clusterId, { clusterId, view: iframe });
   parentElem.appendChild(iframe);
+  logger.info(`[LENS-VIEW]: waiting cluster to be ready, clusterId=${clusterId}`);
+  await cluster.whenReady;
```

Comment:
what happens if the cluster is never ready? We have an async function that never resolves?

Reply:
@jim-docker why it could be never ready? These views are created when you first time opens a cluster, so it should be ready at some point?

Reply:
If there's a bug? I was just concerned that if `InitView()` blocks here because of a bug what will the user see?